docs/streams/upgrade-guide.html

```diff
@@ -84,7 +84,13 @@
         and can be obtained by calling <code>Topology#describe()</code>.
         An example using this new API is shown in the <a href="/{{version}}/documentation/streams/quickstart">quickstart section</a>.
     </p>
-
+    
+    <p>
```

Comment:
Note that we would not need to mention about deprecation StreamsBuilder is newly introduced in 1.0.0. If we managed to get it in 1.0.0 we would only mention "new methods added in KStream: merge" as the "newly added StreamsBuilder no longer have the merge function that KStreamBuilder has".